### PR TITLE
Chore: Icons & typography

### PR DESCRIPTION
**Pull request by WissenIstNacht**
Changes:

- Imported Manrope font 
- Added 'add' and 'close' icons to the respective buttons.
- Improved size/position of certain icons

### CODE REVIEW

File: packages/ui-components/src/components/button/menuButton.tsx [hunk 1]
```diff
@@ -41,7 +41,11 @@ export const MenuButton = ({
     return (
       <OpenButton onClick={onClick} size={size}>
         <FlexDiv side={'left'}>
-          <IconClose />
+          <IconClose
+            width={12}
+            height={12}
+            className="fill-current text-primary-500"
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/button/menuButton.tsx]: Should `fill-current` be passed down? Also, the SVG should automatically take on the text color as well.

File: packages/ui-components/src/styles/tailwind.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600;700;800&display=swap');
```
Comment (sepehr2github) [inline packages/ui-components/src/styles/tailwind.css:1]: I think It's better to use a local font instead of using a remote link. What do you think?

**Review by sepehr2github — COMMENTED** [review]
Thanks.
Looks Great!

**Review by Fabricevladimir — APPROVED** [review]
Looks great!

File: packages/ui-components/src/styles/tailwind.css (hunk 2 above)
Comment (WissenIstNacht) [reply packages/ui-components/src/styles/tailwind.css:1]: Agreed! Can I suggest waiting for the design tokens to come in and tackle this in a corresponding ticket?  (use this as a temporary fix so we don't have to demo serif fonts 😋)

**Review by sepehr2github — APPROVED** [review]

**Pull request merged**